backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -310,11 +310,11 @@ public Response legacyUnlock(@PathParam("vaultId") UUID vaultId, @PathParam("dev
 	@APIResponse(responseCode = "402", description = "number of effective vault users exceeds available license seats")
 	@APIResponse(responseCode = "403", description = "not a vault member")
 	@APIResponse(responseCode = "404", description = "unknown vault")
-	@APIResponse(responseCode = "410", description = "Vault is archived")
+	@APIResponse(responseCode = "410", description = "Vault is archived. Only returned if ignoreArchivedVault is false or not set, otherwise the archived flag is ignored")
 	@ActiveLicense // may throw 402
-	public String unlock(@PathParam("vaultId") UUID vaultId) {
+	public String unlock(@PathParam("vaultId") UUID vaultId, @QueryParam("ignoreArchivedFlag") Optional<Boolean> ignoreArchivedFlag) {
```

Comment:
1. Instead of an `Optional`, use a plain `@DefaultValue("false") boolean`
2. I'd suggest to use a more "fluent" name, such as `unlock [...] despiteArchived` or `evenIfArchived` in the exposed API (internal variable name can be different to match internal logic)

```suggestion
	public String unlock(@PathParam("vaultId") UUID vaultId, @QueryParam("evenIfArchived") @DefaultValue("false") ignoreArchived) {
```

Reply:
Applied in [e959779](https://github.com/cryptomator/hub/pull/230/commits/e959779b5f86fefe6739fa6db7baabc142e14c89)